packages/clickable/index.ts

```diff
@@ -0,0 +1 @@
+export { default as Clickable } from "./clickable";
```

Comment:
Could you explain this level of indirection to me?

Reply:
What do you mean by indirection, do you mean that we export something from a different file? or the default as Clickable?

Reply:
But since I declared the children not to be optional anymore this is actually not needed anymore.

Reply:
I think now I get it :)

Reply:
I did not, can you explain why this file is necessary?

Reply:
This file is necessary to create consistency. 
We have multiple packages like `clickable`. And their components are located in `packages/clickable/clickable.tsx` in this case. We would like to import like this: `packages/clickable` which redirects to the index file. Which is basically exporting all the components. Within this package. 

We also do have the packages/index.js file. Which will be also exporting all the packages. So the user now gets multiple ways to import in the application:
 1. `import {Clickable} from "@dcos/ui-kit"; -- Which imports the complete library. Which is not shakeable even if you only use one component.
2. `import {Clickable} from "@dcos/ui-kit/packages/clickable"; -- Which imports the complete package. Which is not shakeable even if you only use one component, but it is only containing one package.
3. `import {Clickable} from "@dcos/ui-kit/packages/clickable/components/clickable"; -- Which imports only that exact component. Which is kind of the recommended way since this will only include in the build what is needed. 

At some point, we will split up the whole lib into multiple packages and then there would also be a 4 way to import everything which is: `import {Clickable} from "@dcos/ui-kit-clickable";

Reply:
alright.